sensor/kubernetes/sensor/sensor.go

```diff
@@ -158,7 +158,7 @@ func CreateSensor(client client.Interface, workloadHandler *fake.WorkloadManager
 		return nil, errors.Wrap(err, "creating central client")
 	}
 
-	if features.LocalImageScanning.Enabled() && securedClusterIsNotManagedManually(helmManagedConfig) && env.UseLocalScanner.BooleanSetting() {
+	if features.LocalImageScanning.Enabled() && securedClusterIsNotManagedManually(helmManagedConfig) && env.LocalImageScanningEnabled.BooleanSetting() {
```

Comment:
Nit: This assumes the TLS issuer is able to ignore the secrets if those are not owner by sensor. Even though that is documented on `localScannerTLSIssuerImpl.Start`, it would be nice to document that requirement from the client  code by adding a comment here

Reply:
done